.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "voxblox/pybind11"]
```

Comment:
Ah yeah I think this is the issue... Probably remove this one and keep the other.